dcos_launch/platforms/aws.py

```diff
@@ -295,21 +299,6 @@ def delete(self):
         self.stack.delete()
         log.info('Delete successfully initiated for {}'.format(stack_id))
 
-    @retry_boto_rate_limits
-    @property
-    def status(self):
-        return self.stack.stack_status
```

Comment:
I think this pattern was a good choice for stack_name member variable that you tried to add above